examples/todolist/src/lib.rs

```diff
@@ -30,6 +30,17 @@ impl TodoList {
         self.items.push(entry.text)
     }
 
+    fn add_entries(&mut self, entries: Vec<TodoEntry>) {
+        self.items.extend(entries.iter().map(|e| e.text.clone()))
```

Comment:
Something to think about for later, but...I reckon we could figure out how to avoid the `clone` here since you own the vec and all its contents.